jupyterlab_git/handlers.py

```diff
@@ -516,6 +516,15 @@ async def post(self, path: str = ""):
 
         if data["checkout_branch"]:
             if data["new_check"]:
+                #  Need to check if startpoint is valid
+                isStartPointValid = await self.git._get_branch_reference(
+                    data["startpoint"], local_path
+                )
+
+                #  If start point is not valid, we need to make an empty commit
+                if not isStartPointValid:
```

Comment:
Should it be snake_case for consistency?

```suggestion
                is_start_point_valid = await self.git._get_branch_reference(
                    data["startpoint"], local_path
                )

                #  If start point is not valid, we need to make an empty commit
                if not is_start_point_valid:
```

Reply:
Good call, I guess my brain defaults to TS :rofl: